include/engine/api/table_api.hpp

```diff
@@ -65,7 +66,14 @@ class TableAPI final : public BaseAPI
         }
 
         response.values["durations"] =
-            MakeTable(durations, number_of_sources, number_of_destinations);
+            MakeTable(tables.first, number_of_sources, number_of_destinations);
```

Comment:
Let's rename this to `MakeDurationTable` so it is consistent with `MakeDistanceTable`